config/tool_conf.xml

```diff
@@ -317,6 +317,8 @@
       <tool file="phenomenal/w4m/transformation_config.xml" labels="W4M"/>
       <tool file="phenomenal/w4m/batch_correction.xml" labels="W4M"/> 
       <tool file="phenomenal/w4m/NmrNormalization_xml.xml" labels="W4M"/>
+      <tool file="phenomenal/reshape2/melt.xml" labels="reshape2"/>
```

Comment:
I think the `labels` attribute doesn't belong here.  The tool label comes from the tool's xml definition.  Looking at the Galaxy docs it looks like only `labels="updated"` is valid and has a special meaning:
    https://docs.galaxyproject.org/en/master/admin/tool_panel.html#sections-labels

Reply:
Should these be updated across the board? I just followed the convention that was already used in `tool_conf.xml` for the other tools.